test/runtime.js

```diff
@@ -173,22 +206,44 @@ describe('runtime parameters', function() {
         assert.equal(process.sass.binaryUrl.substr(0, URL.length), URL);
       });
     });
-    describe('checking if environment variable overrides package.json', function() {
+    describe('checking if environment variable overrides .npmrc configuration variable', function() {
```

Comment:
The unit tests are now implemented in the way you suggested. It's now much smaller and cleaner without decreasing the coverage. I'm hoping your happy with the current solution. Just tell me if there is anything else to do.